dev/com.ibm.ws.jdbc_fat_v43/publish/servers/com.ibm.ws.jdbc.fat.v43/server.xml

```diff
@@ -44,6 +44,7 @@
   </library>
 
   <javaPermission codebase="${server.config.dir}/drivers/d43driver.jar" className="java.security.AllPermission"/>
+  <javaPermission className="java.lang.RuntimePermission" name="getClassLoader"/>
```

Comment:
codebase is missing - who are we needing to grant this permission to and why?

Reply:
The application needs it due to the unwrap, I can add the war as the codebase

Reply:
Opening an issue for this.